src/converters/cover.ts

```diff
@@ -163,6 +195,19 @@ class CoverHandler implements ServiceHandler {
     callback(null);
   }
 
+  private handleSetTargetHorizontalTilt(value: CharacteristicValue, callback: CharacteristicSetCallback): void {
+    // map value: angle back to target: percentage
+    const target = Math.round((value as number + 90) / 180 * 100);
+    this.accessory.log.warn('tilt characteristic target value: ' + (value as number).toString());
+    this.accessory.log.warn('tilt target percentage: ' + target.toString());
```

Comment:
These log entries should probably be removed.

Reply:
You're absolutely right, this was an oversight. Especially warning level 😅